bin/python/host_table.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
We should move this into config dir. Possibly use YAML format namely to add comment with explanation of purpose of this file.